seata-spring-boot-starter/src/main/java/io/seata/spring/boot/autoconfigure/properties/SeataProperties.java

```diff
@@ -47,6 +47,10 @@
      * Whether use JDK proxy instead of CGLIB proxy
      */
     private boolean useJdkProxy = false;
+    /**
+     * Specifies which bean do not scanner in the GlobalTransactionScanner
```

Comment:
Specifies beans that won't be scanned in the GlobalTransactionScanner

Reply:
done.